// @tianshu/shared/dist/enums/treatment-category.enum.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.TreatmentCategoryLabel = exports.TreatmentCategory = void 0;
var TreatmentCategory;
(function (TreatmentCategory) {
    TreatmentCategory["ACUPUNCTURE"] = "ACUPUNCTURE";
    TreatmentCategory["TUINA"] = "TUINA";
    TreatmentCategory["MOXIBUSTION"] = "MOXIBUSTION";
    TreatmentCategory["CUPPING"] = "CUPPING";
    TreatmentCategory["SCRAPING"] = "SCRAPING";
    TreatmentCategory["PHYSIOTHERAPY"] = "PHYSIOTHERAPY";
    TreatmentCategory["PSYCHOLOGICAL_COUNSELING"] = "PSYCHOLOGICAL_COUNSELING";
    TreatmentCategory["PSYCHOTHERAPY"] = "PSYCHOTHERAPY";
    TreatmentCategory["MEDICATION"] = "MEDICATION";
    TreatmentCategory["SURGERY"] = "SURGERY";
    TreatmentCategory["REHABILITATION"] = "REHABILITATION";
})(TreatmentCategory || (exports.TreatmentCategory = TreatmentCategory = {}));
exports.TreatmentCategoryLabel = {
    [TreatmentCategory.ACUPUNCTURE]: '针灸',
    [TreatmentCategory.TUINA]: '推拿',
    [TreatmentCategory.MOXIBUSTION]: '艾灸',
    [TreatmentCategory.CUPPING]: '拔罐',
    [TreatmentCategory.SCRAPING]: '刮痧',
    [TreatmentCategory.PHYSIOTHERAPY]: '理疗',
    [TreatmentCategory.PSYCHOLOGICAL_COUNSELING]: '心理咨询',
    [TreatmentCategory.PSYCHOTHERAPY]: '心理治疗',
    [TreatmentCategory.MEDICATION]: '药物治疗',
    [TreatmentCategory.SURGERY]: '手术治疗',
    [TreatmentCategory.REHABILITATION]: '康复训练',
};
